source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
They are not _very_ different though, are they? In most cases we are working with unit quaternions in which case they are the same. I agree with the naming confusion though. Right now `conjugate` and `inverse` does the exact same thing.